tests/resources/test-types.json

```diff
@@ -1596,42 +1798,7 @@
         ]
       },
       {
-        "id": "53",
-        "name": "ADR C&U",
```

Comment:
Is this no longer needed? Can't see anything in the ticket relating to ADR C&U either way

Reply:
Found the comment on the story which covers this. Resovling